client/src/pages/private/ParticipantLanding.js

```diff
@@ -242,6 +242,9 @@ export default () => {
           } else if (item.hired?.length > 0) {
             color = '#17d149';
             status = 'Hired';
+          } else if (!item.interested) {
+            color = '#735613';
```

Comment:
Does Hcap have themes?

Reply:
There is a theme (although it's out of date when considering recent designs), rather than either making something inconsistent with previous lines or refactoring the surrounding code I opted to extend the existing pattern. I don't think the effort vs. reward is worth it here considering our timelines